include/ur_api.h

```diff
@@ -292,6 +292,57 @@ typedef struct ur_rect_region_t {
 
 } ur_rect_region_t;
 
+#if !defined(__GNUC__)
+#pragma endregion
+#endif
+// Intel 'oneAPI' USM Import/Release Extension APIs
+#if !defined(__GNUC__)
+#pragma region usm import release(experimental)
+#endif
+///////////////////////////////////////////////////////////////////////////////
+/// @brief Import memory into USM
+///
+/// @details
+///     - Import memory into USM
+///
+/// @returns
+///     - ::UR_RESULT_SUCCESS
+///     - ::UR_RESULT_ERROR_UNINITIALIZED
+///     - ::UR_RESULT_ERROR_DEVICE_LOST
+///     - ::UR_RESULT_ERROR_INVALID_NULL_HANDLE
+///         + `NULL == hContext`
+///     - ::UR_RESULT_ERROR_INVALID_NULL_POINTER
+///         + `NULL == pMem`
+///     - ::UR_RESULT_ERROR_INVALID_CONTEXT
+///     - ::UR_RESULT_ERROR_INVALID_SIZE
+UR_APIEXPORT ur_result_t UR_APICALL
+urUSMImport(
```

Comment:
@kbenzie  @alycm : did we want this to be extensions?

Reply:
Indeed, symbols added in experimental extensions *must* contain *exp* in their names as defined [here](https://oneapi-src.github.io/unified-runtime/core/CONTRIB.html#naming-convention):

> * All functions must use camel case `urObjectActionExp` convention.
> * All macros must use all caps `UR_NAME_EXP` convention.
> * All structures, enumerations, and other types must follow `ur_exp_name_t` name case convention.

So the two entry points added in this PR should be by convention be called `urUSMImportExp` and `urUSMExportExp`.